Springer Books.js

```diff
@@ -148,6 +148,11 @@ function scrapeBook(doc, url) {
 		var edition = ZU.xpathText(doc, '//dt[text()="Edition Number" or text()="Auflage"]/following-sibling::dd[1]');
 		if (edition && edition !== "1") item.edition = edition;
 		
+		var doi = ZU.xpathText(doc, '//dt[text()="DOI"]/following-sibling::dd[1]');
+		if (doi) {
+			item.extra = "DOI: " + doi;
```

Comment:
Since embedded metadata can write unknown fields to Extra as welll, could we add a test for content in Extra here to make sure we're not overwriting anything?
Along these line

```
if (item.extra) {
 if (!item.extra.includes("DOI") {
   item.extra += "\nDOI: " + doi;
}
else {
 item.extra = "DOI: " + doi;
}

Reply:
(+ `}` before the else)